### PR TITLE
French translation update

### PR DESCRIPTION
**Pull request by martypiton**
Add new translations and correct some mistakes

### CODE REVIEW

File: translations/base-fr.yaml [hunk 1]
```diff
@@ -30,59 +30,59 @@ steamPage:
     longText: >-
         [img]{STEAM_APP_IMAGE}/extras/store_page_gif.gif[/img]
 
-        shapez.io is a game about building factories to automate the creation and processing of increasingly complex shapes across an infinitely expanding map.
-        Upon delivering the requested shapes you will progress within the game and unlock upgrades to speed up your factory.
+        shapez.io est un jeu qui consiste à construire des usines pour automatiser la création et la combinaison de formes de plus en plus complexes sur une carte infinie.
+        Lors de la livraison des formes requises vuos progresserez et debloquerez des améliorations pour accelerer votre usine.
```
Comment (A-312) [inline translations/base-fr.yaml:34]:
```suggestion
        Lors de la livraison des formes requises vous progresserez et débloquerez des améliorations pour accélérer votre usine.
```

File: translations/base-fr.yaml [hunk 2]
```diff
@@ -30,59 +30,59 @@ steamPage:
...
 
         [list]
-            [*] Different maps and challenges (e.g. maps with obstacles)
-            [*] Puzzles (Deliver the requested shape with a restricted area / set of buildings)
-            [*] A story mode where buildings have a cost
-            [*] Configurable map generator (Configure resource/shape size/density, seed and more)
-            [*] Additional types of shapes
-            [*] Performance improvements (The game already runs pretty well!)
-            [*] And much more!
+            [*] Différentes cartes at challenges (ex: carte avec obstacles)
```
Comment (A-312) [inline translations/base-fr.yaml:61]:
```suggestion
            [*] Différentes cartes et challenges (ex: carte avec obstacles)
```

File: translations/base-fr.yaml [hunk 3]
```diff
@@ -30,59 +30,59 @@ steamPage:
...
+            [*] Et plus encore!
         [/list]
 
-        [b]This game is open source![/b]
+        [b]Ce jeu est open source![/b]
 
-        Anybody can contribute, I'm actively involved in the community and attempt to review all suggestions and take feedback into consideration where possible.
-        Be sure to check out my trello board for the full roadmap!
+        N'importe qui peut contribuer, je suis activement impliqué dans la communauté et j'essaye de répondre à toutes les suggestions et prendre en compte votre retour d'expérience quand je le peux.
+        Soyez sûr de jeter un coup d'œil à mon tableau Trello pour toute la feuille de route!
```
Comment (A-312) [inline translations/base-fr.yaml:73]:
```suggestion
        Soyez sûr de jeter un coup d'œil à mon tableau de bord Trello pour le suivi du projet et la planification du développement
```

"roadmap", I prefere "planification du développement"

File: translations/base-fr.yaml [hunk 4]
```diff
@@ -30,59 +30,59 @@ steamPage:
     longText: >-
         [img]{STEAM_APP_IMAGE}/extras/store_page_gif.gif[/img]
 
-        shapez.io is a game about building factories to automate the creation and processing of increasingly complex shapes across an infinitely expanding map.
-        Upon delivering the requested shapes you will progress within the game and unlock upgrades to speed up your factory.
+        shapez.io est un jeu qui consiste à construire des usines pour automatiser la création et la combinaison de formes de plus en plus complexes sur une carte infinie.
+        Lors de la livraison des formes requises vuos progresserez et debloquerez des améliorations pour accelerer votre usine.
 
-        As the demand for shapes increases, you will have to scale up your factory to meet the demand - Don't forget about resources though, you will have to expand across the [b]infinite map[/b]!
+        Au vu de l'augmantation des demandes de formes, vous devrez agrandir votre usine pour répondre à la forte demande - Mais n'oubliez pas les ressources, vous drevrez vous étendre au milieu de cette [b]carte infinie[/b]!
```
Comment (A-312) [inline translations/base-fr.yaml:36]:
```suggestion
        Au vu de l'augmantation des demandes de formes, vous devrez agrandir votre usine pour répondre à la forte demande - Mais n'oubliez pas les ressources, vous drevrez vous étendre au milieu de cette [b]carte infinie[/b] !
```

In french, you have to add a space between a word and `!`.

File: translations/base-fr.yaml [hunk 5]
```diff
@@ -30,59 +30,59 @@ steamPage:
...
 
-        Soon you will have to mix colors and paint your shapes with them - Combine red, green and blue color resources to produce different colors and paint shapes with it to satisfy the demand.
+        Bientôt vous devrez mixer les couleurs et peindre vos formes avec - Combinez les ressources de couleurs rouge, verte et bleue pour produire différentes couleurs et peindre les formes avec pour satisfaire la demande.
 
-        This game features 18 progressive levels (Which should keep you busy for hours already!) but I'm constantly adding new content - There is a lot planned!
+        Ce jeu propose 18 niveaux progressifs (qui devraient déjà vous occuper des heures!) mais j'ajoute constamment de nouveau contenus - Il y en a beaucoup de prévus!
```
Comment (A-312) [inline translations/base-fr.yaml:40]:
```suggestion
        Ce jeu propose 18 niveaux progressifs (qui devraient déjà vous occuper des heures !) mais j'ajoute constamment de nouveau contenus - Il y en a beaucoup de prévus !
```

**Comment by martypiton** [comment]
I've modified the version to apply the suggested changes, thanks A-312 I've missed my little errors !

**Comment by tobspr** [comment]
Ty!

**Comment by martypiton** [comment]
No problem !  ;)

**Pull request merged**